.gitleaks.toml

```diff
@@ -0,0 +1,25 @@
+
+[[ rules ]]
+    id = "high-entropy-base64"
+    [ rules.allowlist ]
+        commits = [
+            "48aa02fe3bd6bad0a61ef122acef5fda38920df1",
```

Comment:
I feel like it would make sense for the exceptions to have comments explaining what the false positive was and why it's OK?

Reply:
Added a bunch of additional commentary to the file. Let me know what you think?